lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -1062,6 +1062,16 @@ export class BrowserCacheManager extends CacheManager {
 
         return currentCacheKey;
     }
+
+    getRedirectRequestContext(): string | null {
+        const key = `${Constants.CACHE_PREFIX}.${TemporaryCacheKeys.REDIRECT_CONTEXT}`;
+        return this.getTemporaryCache(key);
+    }
+     
+    setRedirectRequestContext(value: string): void {
+        const key = `${Constants.CACHE_PREFIX}.${TemporaryCacheKeys.REDIRECT_CONTEXT}`;
+        this.setTemporaryCache(key, value, false);
```

Comment:
I think we do want to generate the key
```suggestion
this.setTemporaryCache(TemporaryCacheKeys.REDIRECT_CONTEXT, value, true);
```

Reply:
Done